test/test_eager_transforms.py

```diff
@@ -2057,6 +2052,112 @@ def compute_loss(new_params, buffers, x, y):
 
         self.assertEqual(result_grads, expected_grads)
 
+    def test_maml_omniglot_minimal_repro(self, device):
+        # TODO: there appears to be precision issues for float32
+        dtype = torch.double
+
+        n_way = 5
+        n_inner_iter = 2
+        num_tasks = 2
+
+        net = nn.Sequential(
+            nn.Conv2d(1, 64, 3),
+            nn.BatchNorm2d(64, momentum=1, affine=True, track_running_stats=False),
```

Comment:
This works if it's GroupNorm but fails if it's BatchNorm2d or LayerNorm, both on cpu and cuda